html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <div>Email</div>
+    <input type="email" name="email">
```

Comment:
Hey Fernando, make sure to add labels to each of the inputs as needed with the proper `for` and `id` attributes. Reach out if you have questions!